### DISCUSSION TITLE
Link directly to source document(s) via generated Citations - whether internal data in blob store or external/public data

### DISCUSSION THREAD

**Discussion by lauramartinazure** [microsoft/sample-app-aoai-chatGPT#433] [Ideas]
I want to return the URL to the document(s) from which the citation is generated, and have the user be able to access or download the document from a blob store in Azure, (or point to an external website if this is not internal data). How would I go about doing this?

**Comment by nietarne** [comment]
Want to do the same thing, did you have any luck?

**Comment by enendufrankc** [comment]
This here can help, it help me 

https://github.com/microsoft/sample-app-aoai-chatGPT/discussions/1170
